src/site/_includes/tags-index.njk

```diff
@@ -1,26 +1,36 @@
 ---
-layout: default
+layout: default-next
 ---
 {% from 'macros/tag-card.njk' import tagCard with context %}
-<div>
-  {# Make sure Actions are first in the tab order. #}
-  <div class="w-actions">
+
+<div class="tags">
+  {# @TODO fix subscribe button #}
+  {# <div class="w-actions">
     {% include 'partials/subscribe-action.njk' %}
-  </div>
+  </div> #}
 
-  <div class="w-layout-container w-pb--non">
-    <header class="w-page-header">
-      <h1 class="w-page-header__headline">{{ title }}</h1>
-      <p class="w-page-header__copy">{{ description }}</p>
-    </header>
-  </div>
-  <section class="w-grid">
-    <div class="w-grid__columns w-grid__columns--three" role="list">
+  {# Hero starts #}
+  <header class="region bg-mid-bg">
+    <div class="wrapper">
+      <div class="headline all-center flow">
+        <h2 class="headline__title">{{ title }}</h2>
+        <p>{{ description }}</p>
+      </div>
+    </div>
+  </header>
+  {# Hero ends #}
+
+  <div class="region">
+    {# Tags start #}
+    <div class="wrapper auto-grid">
       {% for tag in paged.elements %}
         {{ tagCard(tag) }}
```

Comment:
This file is identical with shows-index.njk, apart from line 27. Please refactor this and use one card type and one template for all index pages.